app/feature/feature-addon-purchase/src/main/kotlin/com/hedvig/android/feature/addon/purchase/ui/summary/AddonSummaryDestination.kt

```diff
@@ -162,11 +163,21 @@ private fun SummarySuccessScreen(uiState: Content, onConfirmClick: () -> Unit, n
         },
         spaceBetween = 8.dp,
         endSlot = {
-          HedvigText(
-            text = stringResource(
+          val text = if (uiState.totalPriceChange.amount > 0) {
+            // with +
+            stringResource(
               R.string.ADDON_FLOW_PRICE_LABEL,
-              uiState.summaryParameters.quote.price.amount.toInt(),
-            ),
+              uiState.totalPriceChange.amount.toInt(),
+            )
+          } // without + (supposedly with minus)
+          else {
+            stringResource(
+              R.string.TERMINATION_FLOW_PAYMENT_PER_MONTH, // todo: mb better to have a separate key?
+              uiState.totalPriceChange.amount.toInt(),
+            )
+          }
```

Comment:
So it's without minus now, but we do want a minus to exist here I suppose? Help me understand a bit here

Reply:
Asked Julia:)